tests/test_likelihoods.py

```diff
@@ -64,31 +64,24 @@ def __repr__(self):
 likelihood_setups = [
     LikelihoodSetup(Gaussian()),
     LikelihoodSetup(StudentT()),
-    LikelihoodSetup(Beta(),
-                    Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
-    LikelihoodSetup(Ordinal(np.array([-1, 1])),
-                    Y=tf.random.uniform(Datum.Yshape, 0, 3, dtype=default_int())),
-    LikelihoodSetup(Poisson(invlink=tf.square),
-                    Y=tf.random.poisson(Datum.Yshape, 1.0, dtype=default_float())),
-    LikelihoodSetup(Exponential(invlink=tf.square),
-                    Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
-    LikelihoodSetup(Gamma(invlink=tf.square),
-                    Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
-    LikelihoodSetup(Bernoulli(invlink=tf.sigmoid),
-                    Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
-
-    pytest.param(LikelihoodSetup(MultiClass(2),
-                                 Y=tf.argmax(Datum.Y, 1).numpy().reshape(-1, 1),
-                                 rtol=1e-3, atol=1e-3),
+    LikelihoodSetup(Beta(), Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
+    LikelihoodSetup(Ordinal(np.array([-1, 1])), Y=tf.random.uniform(Datum.Yshape, 0, 3, dtype=default_int())),
+    LikelihoodSetup(Poisson(invlink=tf.square), Y=tf.random.poisson(Datum.Yshape, 1.0, dtype=default_float())),
+    LikelihoodSetup(Exponential(invlink=tf.square), Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
+    LikelihoodSetup(Gamma(invlink=tf.square), Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
+    LikelihoodSetup(Bernoulli(invlink=tf.sigmoid), Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
+    pytest.param(LikelihoodSetup(MultiClass(2), Y=tf.argmax(Datum.Y, 1).numpy().reshape(-1, 1), rtol=1e-3, atol=1e-3),
```

Comment:
Would the previous formatting not make it clearer that the last one has different tolerances, and how the choice of Y relates to the likelihood distribution?

Reply:
This is auto formatting, I don't think it is matter for the test to change. Pls, do not force to change it back :)